### PR TITLE
Autonc as default

### PR DESCRIPTION
**Pull request by mcyprian**
I think we should make name convertor based on Automatic Provides the default for Fedora.

### CODE REVIEW

File: pyp2rpm/name_convertor.py [hunk 1]
```diff
@@ -80,7 +80,7 @@ def rpm_versioned_name(cls, name, version, default_number=False):
                     version), '%{{python{0}_pkgversion}}'.format(version))
         return versioned_name
 
-    def rpm_name(self, name, python_version=None):
+    def rpm_name(self, name, python_version=None, pkg_name=False):
         """Returns name of the package converted to (possibly) correct package
            name according to Packaging Guidelines.
         Args:
```
Comment (irushchyshyn) [inline pyp2rpm/name_convertor.py:86]: Nitpick: please document `pkg_name` argument.

**Review by irushchyshyn — COMMENTED** [review]
Looks good, however I find it strange that for Fedora `pyp2rpm <package> --autonc ` and `pyp2rpm <package>` does the same. 
What is the use of the `--autonc` then?

**Comment by mcyprian** [comment]
Documentation of pkg_name argument added. Flag --autonc has no effect only in case fedora template is used. Someone might use other (custom) template with automatic provides based nc. Also in case someone uses --autonc at this moment removing it will brake his scripts. In my opinion we should keep it as it is for now.

**Comment by irushchyshyn** [comment]
Looks good!👍 
Please do the rebase and merge.

**Review by irushchyshyn — APPROVED** [review]

**Comment by mcyprian** [comment]
Thanks ;)

**Pull request merged**